vcluster/_partials/deploy/deploy.mdx

```diff
@@ -173,14 +175,147 @@ Commit and push these files to your configured ArgoCD repository.
 
 Sync your ArgoCD repository with your configured cluster:
 
-<InterpolatedCodeBlock 
- code={`argocd app sync [[VAR:CLUSTER NAME:my-vcluster]]`} 
- language="bash" 
+<InterpolatedCodeBlock
+ code={`argocd app sync [[VAR:CLUSTER NAME:my-vcluster]]`}
+ language="bash"
 />
 
 </Step>
 </Flow>
 
+</TabItem>
+<TabItem value="flux">
+<Flow id="flux-deploy-vcluster">
+<Step>
+Create a Helm repository source in the Git repository that Flux monitors, so Flux can fetch the vCluster Helm charts automatically. In this example, related files are stored under the `clusters` folder:
+
+```yaml title="clusters/sources/vcluster-repository.yaml"
+---
+apiVersion: source.toolkit.fluxcd.io/v1
+kind: HelmRepository
+metadata:
+  name: vcluster
+  namespace: flux-system
+spec:
+  interval: 1h
+  url: https://charts.loft.sh
+```
+
+</Step>
+
+<Step>
+The vCluster will be deployed into the namespace `team-x` of the host cluster. If this namespace doesn’t already exist, create it with:
+```bash title="Create the namespace"
+kubectl create namespace team-x
+```
+
+</Step>
+
+<Step>
+
+Create a vCluster `HelmRelease` file in your Git repository. This `HelmRelease` tells Flux how to deploy a vCluster in your Kubernetes cluster using the configured Helm charts in Step 2:
+
+```yaml title="clusters/production/vcluster-demo.yaml"
```

Comment:
The code blocks should be using the `InterpolatedCodeBlock` component similar to other installation methods.